cmd/logger.go

```diff
@@ -137,6 +155,24 @@ func getSource() string {
 	return fmt.Sprintf("[%s:%d:%s()]", filename, lineNum, funcName)
 }
 
+func printJSON(level, source, cause, msg string, stack []string, data ...interface{}) {
```

Comment:
`level` should be `logrus.Level` and use it as `strings.Title(level.String())` for string.